viewing_party/party.py

```diff
@@ -1,21 +1,125 @@
+# -----------------------------------------
 # ------------- WAVE 1 --------------------
+# -----------------------------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict.update({"title": title})
+        movie_dict.update({"genre": genre})
+        movie_dict.update({"rating": rating})
+        return movie_dict
+    else:
+        return None   
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+    
+
+def watch_movie(user_data, title):
+    for movie_dict in user_data["watchlist"]:
+        if movie_dict["title"] == title:
+            user_data["watched"].append(movie_dict)
+            user_data["watchlist"].remove(movie_dict)
+            return user_data
+    return user_data

```

Comment:
we only need one return statement. But which one? Well, it depends! Do we want to immediately end the function once we've appended and removed one movie? Is there a possibility there could be duplicates of the same movie inside "watched" and "watchlist"? Maybe!

Either one of these works in this context, but do pick only *one*